bessctl/commands.py

```diff
@@ -1817,37 +1818,30 @@ def get_delta(old, new):
 
     def print_header(timestamp, name_len):
         cli.fout.write('\n')
-        fmt = '%-{}s%12s%12s%12s%12s%12s%12s\n'.format(name_len)
-        cli.fout.write(fmt %
-                       (time.strftime('%X') + str(timestamp % 1)[1:8],
-                        'CPU MHz', 'scheduled', 'Mpps', 'Mbps',
-                        'pkts/sched', 'cycles/p'))
+        fmt = '{:<%d}{:>12}{:>12}{:>12}{:>12}{:>12}{:>12}\n' % (name_len,)
```

Comment:
Mixing old formatting with new formatting. I know some disapprove, but it sure is a lot simpler than `'{{:<{:d}}}{{:>12}}...'.format(name_len)`... 😺

Reply:
Yeah, it feels a bit dirty. I agree it's much easier to read than the alternative, though :)